pkg/skaffold/runner/runner.go

```diff
@@ -465,6 +465,12 @@ func DependenciesForArtifact(ctx context.Context, a *latest.Artifact) ([]string,
 	}
 
 	if err != nil {
+		// if the context was cancelled act as if all is well
+		// TODO(dgageot): this should be even higher in the call chain.
+		if ctx.Err() == context.Canceled {
```

Comment:
This will mask *all* errors.  Perhaps you could log the error at a DEBUG level?

Reply:
Well, the idea is to not show annoying errors when the user presses ctrl-c. No matter what the error is

Reply:
That's why I suggested DEBUG :-)  Just in case those errors are due to something other than SIGTERM/KILL.  Anyways, it's not a blocked from me.

Reply:
I'm going to add debug logs